protocol/x/subaccounts/keeper/grpc_query_withdrawal_and_transfers_blocked_info.go

```diff
@@ -0,0 +1,40 @@
+package keeper
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	"github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
+)
+
+func (k Keeper) GetWithdrawalAndTransfersBlockedInfo(
+	c context.Context,
+	req *types.QueryGetWithdrawalAndTransfersBlockedInfoRequest,
+) (*types.QueryGetWithdrawalAndTransfersBlockedInfoResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+	ctx := sdktypes.UnwrapSDKContext(c)
+
+	downtimeInfo := k.blocktimeKeeper.GetDowntimeInfoFor(
+		ctx,
+		types.WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_CHAIN_OUTAGE_DURATION,
+	)
+	chainOutageSeenAtBlock, chainOutageExists := downtimeInfo.BlockInfo.Height, downtimeInfo.BlockInfo.Height > 0
+	negativeTncSubaccountSeenAtBlock, negativeTncSubaccountSeenAtBlockExists := k.GetNegativeTncSubaccountSeenAtBlock(ctx)
+
+	// Withdrawals and transfers are blocked at the
+	withdrawalsAndTransfersBlockedUntilBlock := uint32(0)
```

Comment:
Blocked until block 0 means no blocking right?

Reply:
Yep, essentially means withdrawals and transfers are unblocked at block 0 (not blocked)